route/build/src/write_restart_pio.f90

```diff
@@ -23,16 +25,19 @@ MODULE write_restart_pio
 USE public_var,        ONLY: iulog             ! i/o logical unit number
 USE public_var,        ONLY: integerMissing
 USE public_var,        ONLY: realMissing
+USE public_var,        ONLY: tracer             ! tracer logical
 ! meta data
 USE globalData,        ONLY: meta_stateDims  ! states dimension meta
 USE globalData,        ONLY: meta_qDims
 USE globalData,        ONLY: meta_irf_bas
+USE globalData,        ONLY: meta_bas_tracer
 USE globalData,        ONLY: meta_basinQ
 USE globalData,        ONLY: meta_irf
 USE globalData,        ONLY: meta_kwt
 USE globalData,        ONLY: meta_kw
 USE globalData,        ONLY: meta_mc
 USE globalData,        ONLY: meta_dw
+USE globalData,        ONLY: meta_tracer
```

Comment:
Same here...

Reply:
`meta_tracer` is changed to `meta_solute` and added comments (for all the meta data)